src/components/BlogPost.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Container from '@material-ui/core/Container';
+import Post from './Post';
+import ListOfComments from './ListOfComments';
+
+interface BlogPostProps {
+    title: string;
```

Comment:
⛏️  `readonly title: string` for all fields

Reply:
sick did not know that we can do that. Will do that but this will ultimately just end up taking in an id